maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -835,6 +835,20 @@
 	},
 /turf/open/floor/wood/ship,
```

Comment:
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/113512bc-9fa5-4f80-86e2-31e6b440ad0f)
- Buttons that can be destroyed on indestructible walls. Could be code issue, tldr. drop big bomb and bye buttons.
- Walls that are named "hull" when they are not "hull/indestructible"  Give a quick rename perhaps
- Areas/deadzones are generally bad
- Wyd with 4+ thicccness of hull walls 
- Semiotics should normally be near the doorway or near the object. For example hazard warnings are near doorways leading to the hazard, while if its an object itself near the object. Almayer has good examples overall ;D
- APC can be destroyed if somehow a explosion gets there (events) 
- Dunno why there are vents in deadzones if they are welded off, plus, pipes under walls reeeee


![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/6d52982a-7aa4-4cba-9d66-bfb76150a124)
- Should the retractable railing cover the entire width of entry? 

![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/57ae82a3-dcd2-4480-8adb-6a33e8e9640c)
- Different areas used?

+1 Pretty cool design tho :D (now go fix)